### PR TITLE
fix(sql-execute): commit/rollback button does not sync with trans state

### LINKED ISSUE #2096: [Bug]: commit and rollback button does not sync with transaction status for oceanbase 4.2
### ODC version

4.1.0~4.2.3

### OB version

4.2.1.1~4.2.1.4

### What happened?

 commit and rollback button does not sync with transaction status for oceanbase 4.2

after check flow, rootcause was got transaction state failed.

below query return empty resultset.
```sql
SELECT * FROM SYS.GV$OB_PROCESSLIST WHERE ID=userenv('sessionid')
```

image

### What did you expect to happen?

if set autocommit OFF, commit and rollback button shoudl sync with transaction status 

### How can we reproduce it (as minimally and precisely as possible)?

1. set autocommit OFF
2. execute some DML statement
3. check the commit/rollback button, should clickable
4. do commit
5. check the commit/rollback button, should be gray, not clickable

### Anything else we need to know?

_No response_

### Cloud

_No response_

### PR DESCRIPTION
**Pull request by yizhouxw**
#### What type of PR is this?

type-bug
#### What this PR does / why we need it:
while connec to oceanbase 4.2.1, commit/rollback button does not sync with trans state. 

#### Which issue(s) this PR fixes:

Fixes #2096 

#### Special notes for your reviewer:
odc get transaction state from gv$ob_processlist, 
however it may result into empty while connect to oceanabase 4.2.1.1~4.2.1.4.

change to access `v$ob_processlist` instead of `gv$ob_processlist` will workaround this issue.

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/stats/mysql/OBMySQLNoLessThan400StatsAccessor.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public class OBMySQLNoLessThan400StatsAccessor extends OBMySQLStatsAccessor {
     private static final String OB40_QUERY_ALL_SESSIONS =
             "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.GV$OB_PROCESSLIST";
     private static final String OB40_QUERY_CURRENT_SESSION =
-            "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.GV$OB_PROCESSLIST WHERE `ID`=connection_id()";
+            "SELECT " + OB40_SESSION_COLUMNS + " FROM oceanbase.V$OB_PROCESSLIST WHERE `ID`=connection_id()";
```
Comment (yhilmare) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/stats/mysql/OBMySQLNoLessThan400StatsAccessor.java:59]: OBMySQLNoLessThan400StatsAccessor#OB40_QUERY_ALL_SESSIONS
still use `GV$OB_PROCESSLIST`, is this expected?

Comment (yizhouxw) [reply libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/stats/mysql/OBMySQLNoLessThan400StatsAccessor.java:59]: yes, query all requires all sessions for full tenant.
while current session can only locate at target observer.

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by PeachThinking — APPROVED** [review]
LGTM

**Pull request merged**